tests/test_project/tests/test_qa.py

```diff
@@ -44,5 +44,54 @@ def test_qa_call_check_migration_name_failure(self):
                 else:
                     self.fail('SystemExit or Exception not raised')
 
+    def test_qa_call_check_commit_message_pass(self):
+        options = [
+            'commitcheck',
+            '--message',
+            "[qa] Updated more file and fix "
+            "problem #20\n\nAdded more files Fixes #20"
```

Comment:
These are all good commit messages, use these ones as starting point

```
[qa] Minor clean up operations
```
(it's ok to not reference any issue sometimes)

```
[qa] Improved Y #2

Related to #2
```
(but if an issue is mentioned it should be specified whether the commit fixes it or just references it)

```
[qa] Finished task #2

Closes #2
Related to #1
```
(multiple references can be added in the extended desc)

These commit messages must not fail.